packages/forma-36-codemod/transforms/v4-badge.js

```diff
@@ -0,0 +1,56 @@
+const {
+  getComponentLocalName,
+  renameProperties,
+  changeProperties,
+  changeImportWithComponentName,
+  changeComponentName,
+} = require('../utils');
+const { getFormaImport, shouldSkipUpdateImport } = require('../utils/config');
+
+module.exports = function (file, api) {
+  const j = api.jscodeshift;
```

Comment:
What do you think about renaming this from `j` to `codeshift`?

I'm a pro descriptive names person hahaha

Reply:
sure, of course. I would do it globally than in the whole package. Maybe separate PR?

Reply:
`j` is a common name in all codeshift examples you can find on Github

Reply:
got it!
Nevermind then